tests/hw2/task1_test.py

```diff
@@ -0,0 +1,53 @@
+import os
+import tempfile
+
+import pytest
+
+from homework2.task1 import for_text
+
+
+@pytest.fixture()
+def temp_file() -> str:
+    text = "фафаа\n\u006e\u0072\n--\nssuummeerr\naa\naaaabbbbcccc\n))\n!!\n..\n,,\nr)\n"
+    temp_dir = tempfile.gettempdir()
+    temp_file = f"{temp_dir}/task1_file1.txt"
+    with open(temp_file, "w") as fw:
+        fw.write(text)
+    return temp_file
```

Comment:
Файл останется в файловой системе после заверешения тестов. Но фикстуры позволяют легко выполнять действия после тестов, нужно только вместо `return` использовать `yield` и после него поместить код который будет выполнен по окончании теста:
```suggestion
    yield temp_file
    # тут код для удаления temp_file после тестов
```